build/scripts/install_tarball

```diff
@@ -2,7 +2,33 @@
 
 set -eu
 
+# Usage instruction for the install_tarball script
+usage () {
+    cat <<EOF
+Usage: $0 [OPTIONS]
+    The following options may be given :
+        -u                  Update PMM-Agent
+        -help) usage ;;
+Example:
+        For help command - $0 -help
+        For PMM tarball installation - $0
+        To update the installed PMM tarball - $0 -u
+EOF
+        exit 1
+}
+
+UPDATE=0
+for arg in "$@"
+do
+    case "$arg" in
+        "-u") UPDATE=1 ;;
+        "-help") usage ;;
+        *) echo "Invalid option:$arg"; usage;;
+    esac
+done
+
 CURRENT_DIR="$(pwd)"
+PMM_VER=$(echo "$CURRENT_DIR" | cut -d'-' -f3)
```

Comment:
Why do we assume PMM version can be taken from current directory? What if the use runs the whole package to an arbitrary directory and then runs `./install_tarball`?

IMO `PMM_VER` will be an empty string at the very best.

Reply:
According to the installation guide https://docs.percona.com/percona-monitoring-and-management/setting-up/client/index.html#binary-package, we run the install_tarball script from the extracted PMM tarball directory (eg: pmm2-client-2.41.1), hence I fetched PMM_VER using this.

Also PMM_VER isn't really necessary. I have just used it to print a message when the installation is successful. We can remove the PMM_VER variable. Not sure if there is any other way to find the version.

Reply:
> According to the installation guide https://docs.percona.com/percona-monitoring-and-management/setting-up/client/index.html#binary-package, we run the install_tarball script from the extracted PMM tarball directory (eg: pmm2-client-2.41.1) which is the current directory, hence I fetched PMM_VER using this.

I don't think this answers the question whether or not the user won't run the install script from an arbitrary directory :)

Reply:
> Also PMM_VER isn't really necessary. I have just used it to print a message when the installation is successful. We can remove the PMM_VER variable. Not sure if there is any other way to find the version.

It should be possible to run `cat VERSION` from the directory where the user unpacks the tarball (see attachement). But that file is empty now for some reason, i.e. its length is just one byte :( FYI @talhabinrizwan 

<img width="916" alt="Screenshot 2024-02-28 at 23 50 13" src="https://github.com/percona/pmm/assets/81549/37ea4758-d75a-444d-82b6-0c0c8e06d549">

Reply:
I suggest we skip printing the version for now.

Reply:
Yes, thought of fetching from the VERSION file but found it empty. Sure, I shall remove "PMM_VER".